api/capi/include/nnstreamer.h

```diff
@@ -531,28 +618,28 @@ void ml_util_free_tensors_info (ml_tensors_info_s *info);
 void ml_util_free_tensors_data (ml_tensors_data_s **data);
 
 /**
- * @brief Allocates a tensor data frame with the given tensors type.
+ * @brief Allocates a tensor data frame with the given tensors information.
  * @since_tizen 5.5
- * @param[in] info The tensors information for the allocation.
+ * @param[in] info The handle of tensors information for the allocation.
  * @return @c Tensors data pointer allocated. Null if error. Caller is responsible to free the allocated data with ml_util_free_tensors_data().
  * @retval NULL There is an error. Call ml_util_get_last_error() to get specific error code.
  */
-ml_tensors_data_s *ml_util_allocate_tensors_data (const ml_tensors_info_s *info);
+ml_tensors_data_s *ml_util_allocate_tensors_data (const ml_tensors_info_h info);
 
 /**
  * @brief Checks the availability of the given execution environments.
  * @since_tizen 5.5
+ * @details If the function returns an error, @a available is not changed.
```

Comment:
Please fix it as below. 

```cpp
* @details If the function returns an error, @a available is not changed.
* @since_tizen 5.5
```

For your information, correct tag order is as below.
* Correct tag order:
  @deprecated (mandatory if applicable)
  @platform or @partner
    If @privlevel is 'platform' or 'partner', respectively.
    Should not be used in other cases.
  @brief (mandatory)
  @details
  @since_tizen (mandatory)
  @privlevel
  @privilege (privlevel/privilege can be repeated if necessary)
  @remarks
  @param
  @return (mandatory if return type is not void)
  @retval
  @exception
  @pre
  @post
  @see

Reply:
I updated this.